src/app/public/modules/chevron/chevron.component.scss

```diff
@@ -55,3 +55,34 @@
     transform: rotate(-45deg);
   }
 }
+
+@include sky-theme-modern {
+  .sky-chevron {
+    height: 19px;
+    width: 16px;
+  }
+
+  .sky-chevron-part {
+    background: darken($sky-text-color-icon-borderless, 20%);
```

Comment:
where did this darken come from vs using the borderless icon color straight up?

Reply:
Oops, that was a mistake. Before I make the change...what should this be? Just `$sky-text-color-icon-borderless;` ?

Reply:
i believe that is correct. it should function as if it was a borderless icon button

Reply:
which, we also need to make sure it has the correct hover, focus and active states toward that end

Reply:
@Blackbaud-ToddRoberts updated to act like a borderless icon button in modern theme. Should have the focus/hover/etc states.